ehitchhiking-ui/src/app/shared/components/dialog-list/dialog-list.component.sass

```diff
@@ -0,0 +1,29 @@
+@import "../../../../styles/colors"
+@import "../../../../styles/mixins"
+@import "../../../../styles/fonts-constants"
+@import "../../../../styles/constants"
+
+
+.modal-dialog
+  border: 1px solid $gray
+  display: flex
+  padding: 0
+  overflow-x: scroll
+  .image
+    border-radius: 50%
+    width: 3rem
+    height: 3rem
+  .dlg-content
+    display: flex
+    flex-direction: column
+    flex-wrap: wrap
+    margin-left: $default-offset
+    .dlg-title
+      .dlg-title-text
+        margin: 0.5rem
+        color: #343434
```

Comment:
use one of the existing constants or create a new one

Reply:
done